controllers/vpcendpoint/helpers_test.go

```diff
@@ -34,68 +34,6 @@ import (
 	"k8s.io/client-go/tools/record"
 )
 
-func TestVpcEndpointReconciler_parseClusterInfo(t *testing.T) {
```

Comment:
This removal looks dubious haha, but this test really should have been removed earlier. It's covered by https://github.com/openshift/aws-vpce-operator/blob/main/controllers/vpcendpoint/validation_test.go#L41-L56 now